BEIMA.Client/src/shared/NavBar.js

```diff
@@ -0,0 +1,35 @@
+import logo from './BSU-logo.png';
+import { Row, Col} from 'react-bootstrap';
+import './shared.css';
+
+const NavBar = () => {
+    return (
+        <div className="sharedNavBar">
+            <Col>
+                <Row className="logo"><img src={logo} className="bsu-logo" alt="bsu-logo" /></Row>
+                <Row><div className="sharedHeader">BEIMA</div></Row>
+                <Row>
+                    <Col className="pageLinks">
+                        <Row className="pageLink">
+                            <a href="Devices" className="sharedText">Devices</a>
+                        </Row>
+                        <Row className="pageLink">
+                            <a href="Device Types" className="sharedText">Device Types</a>
+                        </Row> 
+                        <Row className="pageLink">
+                            <a href="Add Device" className="sharedText">Add Device {"\xa0\xa0"}+</a>
```

Comment:
Instead of putting "+" at the end, let's try and put it at the beginning. I envision that all of these sidebar links will have their own unique icon, so it will look a lot more normal in the future. This also avoids having to use nonbreaking spaces.